src/test/java/org/squbs/sample/http/SampleHttpSvcTest.java

```diff
@@ -0,0 +1,87 @@
+package org.squbs.sample.http;
+
+import akka.http.javadsl.Http;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.model.HttpResponse;
+import akka.http.javadsl.model.StatusCodes;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import com.typesafe.config.ConfigFactory;
+import org.junit.Test;
+import org.squbs.testkit.japi.AbstractCustomTestKit;
+import org.squbs.unicomplex.JMX;
+import org.squbs.unicomplex.UnicomplexBoot;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+
+import static org.junit.Assert.assertTrue;
+
+public class SampleHttpSvcTest extends AbstractCustomTestKit {
+
+    final Materializer materializer = ActorMaterializer.create(system());
+
+    public SampleHttpSvcTest() {
+        super(TestConfig.boot);
+    }
+
+    @Test
+    public void testGetSampleHttpSvcNegative() throws Exception {
+        final CompletionStage<HttpResponse> responseFuture =
+            Http.get(system())
+                .singleRequest(HttpRequest.create("http://localhost:" + port() + "/nonExistentContext"), materializer);
+        assertTrue(responseFuture.toCompletableFuture().get().status() != StatusCodes.OK);
+    }
+
+    @Test
+    public void testGetSampleHttpSvcNegWithSpace() throws Exception {
+        final CompletionStage<HttpResponse> responseFuture =
+            Http.get(system())
```

Comment:
I assume we use the actual client instead of testkit is because of https://github.com/paypal/squbs/issues/424 ?